public/javascripts/widgets/html.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Create a HTML element string.
+ *
+ * @param {string} type - The type of the HTML element.
+ * @param {string} id - The ID of the HTML element.
+ * @param {array<string>} classes - The classes of the HTML element.
+ * @param {string} content - The content of the HTML element.
+ * @param {object} extra - The extra content of the element, that the key is the
+ * data or another structure and the value.
+ * @param {boolean} end - If the element should be closed.
```

Comment:
Deixo mais uma vez meu desejo de não ter uma classe dessas, por ser abstração demais para os nossos casos - muito boilerplate pra pouco benefício. Mas se for usar, no mínimo tem que dar os nomes certos aos bois:
```suggestion
 * @param {string} tagName - The type of the HTML element.
 * @param {string} id - The ID of the HTML element.
 * @param {array<string>} classes - The classes of the HTML element.
 * @param {string} innerHTML - The content of the HTML element.
 * @param {object} attributes - The extra content of the element, that the key is the
 * data or another structure and the value.
 * @param {boolean} tagNeedsClosing - If the element should be closed.
```